fairlearn/metrics/_metric_frame.py

```diff
@@ -506,6 +506,9 @@ def overall_ci(
         The elements of the list are indexed by the `ci_quantiles` array supplied
         to the constructor.
         """
+        if self._ci_quantiles is None:
```

Comment:
I'm using this as a proxy to check if the [ci-related parameters were not `None`](https://github.com/fairlearn/fairlearn/blob/main/fairlearn/metrics/_metric_frame.py#L284-L304). I can also just check if the `"overall_ci"` key is in the `_result_cache` dictionnary. I don't have any strong preference

Reply:
I think it would actually be nice to store the oroginal parameters of the metric frame in the object, and check them directly

Reply:
Done @adrinjalali :+1: 

However the pipeline keeps failing when trying to [fetch the dataset from openml](https://github.com/fairlearn/fairlearn/blob/main/test/unit/adversarial/test_preprocessing.py#L36). 

Is the test known to be flaky ?

Reply:
yes that test is unfortunately very much known to be flaky :(

Reply:
Thank you @hildeweerts it's all good now :ok_hand: